### ISSUE TITLE
aanpassing indicator horizontale structuur bij habitat 2310 (versie3)

### ISSUE THREAD

**Issue by ToonHub** [inbo/LSVI#230]
De naam van de voorwaarde zou gewijzigd moeten worden van 'bedekking open vegetaties' naar  'bedekking open vegetaties en kaal zand'.

Ook moet 'open zand' worden toegevoegd aan de studiewaarden. (Liefst 'open zand' ipv 'kaal zand' voor consistentie met andere indicatoren waar ook 'open zand' wordt gebruikt)

**Comment by Patrikoosterlynck** [comment]
Oke Toon, kan ik aanpassen maar van thuis uit via VPN kan ik om een of andere reden niet meer aan de CsLocal databank... wel vanuit HT Brussel, ik kan morgen eventueel daarvoor afkomen ofwel wordt dat dinsdag volgende week standaard zowieso. Zeg maar wat beste is voor jouw voortgang

**Comment by ToonHub** [comment]
Het is niet dringend. Ik kan het probleem eenvoudig omzeilen. OK als je het volgende week dinsdag aanpast.

**Comment by Patrikoosterlynck** [comment]
Naam voorwaarde is gewijzigd naar naar 'bedekking open vegetaties en **kaal** zand' 
of heb je hier ook liever een aanpassing naar 'bedekking open vegetaties en **open** zand'?

"open zand" is toegevoegd aan de studiewaardenlijst

@ElsLommelen ter info, zie bovenstaande aanpassingen omdat ik niet zeker ben wat de volledige implicaties zijn van mijn aanpassingen in de verdere workflow naar package toe

**Comment by ElsLommelen** [comment]
Ja, wacht eens, ik zie mogelijk 1 klein probleempje opduiken: er is ergens een voorwaarde 'aandeel éénjarigen + open zand' waar csv-matig een studiegroep '50open zand' aan toegevoegd wordt.  Als jij die waarde 'open zand' bij de studiegroepen aangepast hebt, zou het kunnen dat die niet meer herkend wordt.  (Ik vermoed dat het over het geval van issue #231 gaat, maar dan vrees ik dat die csv niet meer echt up-to-date is.)

**Comment by Patrikoosterlynck** [comment]
ter info: die aanpassing is niet gebeurd bij 2330_dw (issue #231) noch 2330_bu, wel voor 2310

**Comment by ElsLommelen** [comment]
Vraagje in verband hiermee: ook bij 2330_bu staat een voorwaarde 'bedekking open vegetaties'.  Zal deze ook aangepast worden, of niet?
(Ik vraag dit omdat op het moment de voorwaarde 'bedekking open vegetaties en kaal zand' niet correct naar een analysevariabele vertaald wordt, en ik graag wil weten of ik voor beiden een vertaling moet voorzien, of dat het overal aangepast zal worden.)

**Comment by ToonHub** [comment]
De voorwaarde 'bedekking open vegetaties' bij 2330_bu moet niet aangeapst worden.

De naam 'bedekking open vegetaties en kaal zand' bij 2310 liefst behouden, want zo staat het ook in LSVI-rapport.

**Comment by ElsLommelen** [comment]
Ok, bedankt voor je reactie, dan los ik het zo op.

**Comment by ElsLommelen** [comment]
Dit is in orde in de brondatabank en zal bij een update van de SQLite-databank in het package aangepast worden.
